src/main/kotlin/org/arend/intention/SelfTargetingIntention.kt

```diff
@@ -30,9 +30,9 @@ abstract class SelfTargetingIntention<T : PsiElement>(
     final override fun getText() = text
     final override fun getFamilyName() = familyName
 
-    abstract fun isApplicableTo(element: T, caretOffset: Int, editor: Editor?): Boolean
+    abstract fun isApplicableTo(element: T, caretOffset: Int, editor: Editor): Boolean
 
-    abstract fun applyTo(element: T, project: Project?, editor: Editor?)
+    abstract fun applyTo(element: T, project: Project, editor: Editor)
```

Comment:
I made these not-null because these arguments are never null when these methods are invoked.
By doing this, some null-checks are removed for free.

Reply:
@sxhya